docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/ImageClassification.cs

```diff
@@ -111,11 +115,18 @@ class OutputScores
             public float[] output { get; set; }
         }
 
-        private static string Download(string baseGitPath, string dataFile)
+        private static async Task<string> Download(string baseGitPath, string dataFile)
```

Comment:
Related?  or should 6476 be closed?

https://github.com/dotnet/machinelearning/pull/6476/files

Reply:
We want the deeplearning overview, but the actual download code isn't right. I'll edit that pr to just be the doc then merge it.